LEGO1/mxmidimanager.cpp

```diff
@@ -0,0 +1,159 @@
+#include "mxmidimanager.h"
+#include "mxomni.h"
+#include <Windows.h>

```

Comment:
Minor thing, but so far the rest of the codebase included this header written as `#include <windows.h>`